packages/block-library/src/navigation/style.scss

```diff
@@ -55,6 +57,16 @@
 			&:last-of-type > a {
 				padding-right: 0;
 			}
+
+			// Make submenu icon inherit current text color
+			& > a svg {
+				fill: currentColor;
+			}
+
+			// Add some space between link text & submenu icon
+			& > a > span + span {
```

Comment:
Does this (and the one above it) need to be this specific?

Reply:
Good question, and I actually think that those styles belong in `NavigationLink` styles, not here (`Navigation`). Will update!

Reply:
Good question! I guess it depends on how it's overridden (or replaced?) by theme styles later on. AFAIK, there are no theme-specific styles for this block yet, so I'll investigate and make sure if it's better to do it like this or just use the class-names instead.

Reply:
I would imagine use a class name, otherwise if someone changes the markup at all, these styles won't apply anymore.

Reply:
Also, the specificity of these selectors is pretty high, which would make them harder to be overridden by theme styles. I'd make use of those BEM classes we already have there and flatten it all the way down 😄

Reply:
Because related (design) work is happening in #19681, I don' think that addressing the styling in this PR would be a good idea due to doubling the effort. Let's focus on the functionality here and move the remaining (visual) work to #19681 (or to subsequent PRs).